### PR TITLE
Enable USD using Unity's USD package.

### PR DESCRIPTION
**Pull request by TimAidley**
This adds USD support back in, using the official Unity USD package. Tested on Windows 10.

### CODE REVIEW

File: Assets/Resources/OVRPlatformToolSettings.asset [hunk 1]
```diff
@@ -0,0 +1,22 @@
+%YAML 1.1
```
Comment (mikeage) [inline Assets/Resources/OVRPlatformToolSettings.asset:1]: I think this one doesn't belong in git; it's now in gitignore and appeared with the move to the newest Oculus unity package. Ditto for the .meta associated

Comment (TimAidley) [reply Assets/Resources/OVRPlatformToolSettings.asset:1]: Ah! Good catch!

**Comment by TimAidley** [comment]
I have removed those files.

**Comment by mikeage** [comment]
Looks like #32 introduced a conflict here

**Comment by TimAidley** [comment]
Okay, I've resolved the conflict.

File: Assets/Scripts/Rendering/UsdPathSerializer.cs [hunk 2]
```diff
@@ -90,9 +91,9 @@ class UsdCameraXformSample : XformSample {
     var sketchRoot = ExportUsd.CreateSketchRoot();
     m_UsdCamera = new UsdCameraXformSample();
     try {
-      m_Scene = USD.NET.Scene.Create();
+      m_Scene = USD.NET.Scene.Create(path);
```
Comment (mikeskydev) [inline Assets/Scripts/Rendering/UsdPathSerializer.cs:94]: This throws an Argument Null Exception when attempting to record a video.

File: Assets/Scripts/Rendering/VideoRecorderUtils.cs [hunk 3]
```diff
@@ -153,7 +154,7 @@ static public class VideoRecorderUtils {
                   Path.ChangeExtension(filePath, "usda") : null;
```
Comment (mikeskydev) [inline Assets/Scripts/Rendering/VideoRecorderUtils.cs:154]: This is where the path variable is being set, and where the null is probably coming from.

**Review by mikeskydev — CHANGES_REQUESTED** [review]
Had some issues relating to recording, very similar to those that blocked USD in #2

File: Assets/Scripts/Rendering/UsdPathSerializer.cs (hunk 2 above)
Comment (TimAidley) [reply Assets/Scripts/Rendering/UsdPathSerializer.cs:94]: Ah yes... I forgot it could be null for an unsaved file.

**Review by mikeskydev — APPROVED** [review]

**Comment by mikeskydev** [comment]
Thanks!

**Pull request merged**